src/providers/token-properties-provider.ts

```diff
@@ -89,6 +90,16 @@ export class TokenPropertiesProvider implements ITokenPropertiesProvider {
     for (const address of addressesRaw) {
       const cachedValue = tokenProperties[address];
       if (cachedValue) {
+        metric.putMetric("TokenPropertiesProviderBatchGetCacheHit", 1, MetricLoggerUnit.Count)
+        const tokenFee = cachedValue.tokenFeeResult;
+        const tokenFeeResultExists: BigNumber | undefined = tokenFee && (tokenFee.buyFeeBps || tokenFee.sellFeeBps)
+
+        if (tokenFeeResultExists) {
+          metric.putMetric(`TokenPropertiesProviderCacheHitTokenFeeResultExists${tokenFeeResultExists}`, 1, MetricLoggerUnit.Count)
```

Comment:
do you know how this metric will look like in practice?

Reply:
Yep, I'm judging from `TokenPropertiesProviderCacheMissTokenFeeResultExists` from the screenshot, it will look like `TokenPropertiesProviderCacheHitTokenFeeResultExists0`, `TokenPropertiesProviderCacheHitTokenFeeResultExists300`, etc. The trick here as long as total possibility of feeBps is not crazily high, we won't hit metrics cardinalities under the single namespace in routing-api.